#63 - Add past data to meetups files

paulinaszkudlarek opened the pull request:
This should close #63 

Added links to YouTube and Slideshare to presentations.
Added company Polcode

EwelinaLasowy reviewed (commented):
1. When I run command `docker-compose exec php php backend/builder.php` I see some warning in the console:

![image](https://user-images.githubusercontent.com/56546832/226607342-f1fcc7a2-3482-4cb5-b095-31f0a96d5878.png)

2. On the last tag I will remove comma

![image](https://user-images.githubusercontent.com/56546832/226607700-3e8199ce-4592-469d-8560-83a5dafbeab9.png)

Or maybe we should make tags as badges? 

What should happen if I click tag? For now nothing happens.

3. Why preview of presentations is the same for different people? But when I click "Zobacz na Slideshare" it redirects for correct presentation.

![image](https://user-images.githubusercontent.com/56546832/226608387-7aad41b2-fc7c-4d7a-83c4-963de8d69e38.png)

paulinaszkudlarek commented:
1. Fixed 
2. It will be changed in #72 
3. It will be also changed in #72 

 I would let stay PR #71 only for static data, and in #72 change design, links to slideshare, etc.
